### ISSUE TITLE
Error on https://ruffle.rs/demo/

### ISSUE THREAD

**Issue by unblocked-games-premium** [ruffle-rs/ruffle#6695]
# Error Info
Error name: Error
Error message: panicked at 'called `Option::unwrap()` on a `None` value', core/src/display_object/container.rs:853:18
Error stack:
```
Error: panicked at 'called `Option::unwrap()` on a `None` value', core/src/display_object/container.rs:853:18
    at c.wbg.__wbg_new_df6e6ab7a65c4c4d (https://ruffle.rs/demo/966.index.js:1:7197)
    at std::sync::once::Once::call_once::{{closure}}::he6e984b98bfd4469 (https://ruffle.rs/demo/684c378ac33688c7abc2.wasm:wasm-function[1060]:0x2b1ec6)
    at std::sync::once::Once::call_inner::h07d0245bef259c27 (https://ruffle.rs/demo/684c378ac33688c7abc2.wasm:wasm-function[1504]:0x32b01a)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::ha12d7190a4c11fdc (https://ruffle.rs/demo/684c378ac33688c7abc2.wasm:wasm-function[5082]:0x48fa42)
    at std::panicking::rust_panic_with_hook::h623ac32ff431b114 (https://ruffle.rs/demo/684c378ac33688c7abc2.wasm:wasm-function[3347]:0x42c7f8)
    at core::panicking::panic_fmt::h5118e89563022e7e (https://ruffle.rs/demo/684c378ac33688c7abc2.wasm:wasm-function[5171]:0x496df3)
    at core::panicking::panic::he88f1b1582b09c8c (https://ruffle.rs/demo/684c378ac33688c7abc2.wasm:wasm-function[5080]:0x48f5d5)
    at core::ops::function::FnOnce::call_once::h2467d7955bae721a (https://ruffle.rs/demo/684c378ac33688c7abc2.wasm:wasm-function[1139]:0x2c8a31)
    at ruffle_core::avm1::function::Executable::exec::hedc1751271333af2 (https://ruffle.rs/demo/684c378ac33688c7abc2.wasm:wasm-function[431]:0x1387cf)
    at <ruffle_core::avm1::object::Object as ruffle_core::avm1::object::TObject>::call_method::hbd8f85b95d733bff (https://ruffle.rs/demo/684c378ac33688c7abc2.wasm:wasm-function[420]:0x12629e)
```

# Player Info
Allows script access: false

# Page Info
Page URL: https://ruffle.rs/demo/

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.75 Safari/537.36
OS: Win32

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-04-12
Channel: nightly
Built: 2022-04-12T00:15:51.044Z
Commit: 24f00238a77021bea392e0a91c2125ff6e5d7430
Is extension: false

game: electricman 2 HS

**Comment by relrelb** [comment]
Duplicate of #4932.
